src/javascripts/Views/recordView.js

```diff
@@ -24,7 +24,11 @@ ngapp.controller('recordViewController', function($scope, $element, $timeout, ht
 
     $scope.toggleAddressBar = function(visible) {
         $scope.showAddressBar = visible;
-        visible ? $timeout($scope.focusAddressInput, 50) : $scope.treeElement.focus();
```

Comment:
does this fix some kind of bug with the record view?  not sure about the motivation for this change (seems outside of the scope of the PR)?